lighthouse-cli/test/smokehouse/test-definitions/dobetterweb/dbw-expectations.js

```diff
@@ -11,6 +11,10 @@
  */
 const expectations = {
   networkRequests: {
+    // Number of network requests differs between Fraggle Rock and legacy modes because
+    // FR has fewer passes, preserve this check in legacy mode only.
+    _legacyOnly: true,
```

Comment:
should this be updated and be `_fraggleRockOnly`? It took us a while to figure out the re-requests and seems like it might be important to catch FR runner regressions (re: #11506)?

Reply:
sure I'll do it that way instead 👍